new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful and popular editor that has many commands. Unlike most
+text editors where you use the keyboard to enter text and arrow keys to move you
+cursor, Vim uses the keyboard to enter commands to manipulate the text.
```

Comment:
Yes, Vim does not enter commands, that's stupid. About the other thing, I don't think we need to put focus on Vim being *different*, rather Vim being *better*. How about this?

>These commands allow you to express complex thoughts in just a few
> keystroke to move through text and manipulate it much faster than in most
> text editors.